src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_Projections.vb

```diff
@@ -114,13 +114,14 @@ class C
 
                 state.SendTypeCharsToSpecificViewAndBuffer(" ", view, buffer)
                 Await state.WaitForAsynchronousOperationsAsync()
-                Await state.AssertSelectedCompletionItem(displayText:="string", isHardSelected:=True)
+                Await state.AssertSelectedCompletionItem(displayText:="string", isHardSelected:=True, projectionsView:=view)
             End Using
         End Function
 
         <WorkItem(771761, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/771761")>
-        <MemberData(NameOf(AllCompletionImplementations))>
-        <WpfTheory(Skip:="https://github.com/dotnet/roslyn/issues/24846"), Trait(Traits.Feature, Traits.Features.Completion)>
+        <InlineData(CompletionImplementation.Modern)>
+        <InlineData(CompletionImplementation.Legacy, Skip:="https://github.com/dotnet/roslyn/issues/24846")>
```

Comment:
i can see this was skipped before.  but we really need to ensure that these skipped tests are unskipped so we validate behavior.  This behavior in particular is critical to preserve.  Note: it's unclear to me what happened here.  This test was skipped back in february, but was never assigned to anyone.  Disabled tests used to be high priority to fix.  Tagging @jinujoseph for visibility on this.

Reply:
Just a note, we're enabling the test in the new completion. I'm resolving this issue because I do not see any action for it in the PR. 

---
In reply to: [232508529](https://github.com/dotnet/roslyn/pull/29016#discussion_r232508529) [](ancestors = 232508529)